tests/1_unit-tests.js

```diff
@@ -1,69 +1,52 @@
-var chai = require('chai');
-var assert = chai.assert;
+const chai = require('chai');
+const assert = chai.assert;
 
 suite('Unit Tests', function(){
   
   // Make ALL tests pass
   // !! Don't scramble the Assertions. We rely on their order to check the results !!
   suite('Basic Assertions', function() {
-    /** assert.fail() will always fail. Change it into something more useful... **/
-  
-    /** 1 - Use assert.isNull() or assert.isNotNull() to make the tests pass. **/
     test('#isNull, #isNotNull', function(){
       assert.fail(null, 'this is an optional error description - e.g. null is null');
       assert.fail( 1, '1 is not null');
```

Comment:
```suggestion
      assert.fail(null);
      assert.fail(1);
```

There's a lot of text in these messages, wondering if it would clean things up to remove them. Some of them are certainly kind of helpful. What do you think @Sky020? They could possibly be moved over to /learn if the hint is helpful enough.

Reply:
In terms of this, I would appreciate your input @RandellDawson ?

Reply:
@Sky020 I spoke with @moT01 today and we actually concluded it is fine to leave the `assert` message argument text the way it is now.

The biggest change we can make for the `1_unit-tests.js`, is to be able to remove all the comments that would need translation.  I am proposing something like the following for this first 4 challenges of this section. I have added an example for how to reference one of these commented numbers [in this suggested change to the first challenge's file]() 

```js
suite('Unit Tests', function(){
   suite('Basic Assertions', function() {
    // #1 
    test('#isNull, #isNotNull', function(){
      assert.fail(null, 'this is an optional error description - e.g. null is null');
      assert.fail( 1, '1 is not null');
    });
    // #2
    test('#isDefined, #isUndefined', function(){
      assert.fail( null, 'null is not undefined');
      assert.fail( undefined, 'undefined IS undefined');
      assert.fail( 'hello', 'a string is not undefined' );
    });
    // #3
    test('#isOk, #isNotOk', function() {
      assert.isNotOk(null, 'null is falsey');
      assert.isOk("I'm truthy", 'a string is truthy');
      assert.isOk(true, 'true is truthy');
    }); 
    // #4
    test('#isTrue, #isNotTrue', function(){
      assert.fail( true, 'true is true');
      assert.fail( !!'double negation', 'double negation of a truthy is true');
      assert.fail({ value: 'truthy' }, 'A truthy object is NOT TRUE (neither is false...)' );
    });
  });
  
  // -----------------------------------------------------------------------------
```